tests/shouldwork/TopEntity/TwoTops.hs

```diff
@@ -0,0 +1,10 @@
+module TwoTops where
+import Clash.Prelude
+
+-- https://github.com/clash-lang/clash-compiler/issues/1139
```

Comment:
I actually really like the way Christiaan's being naming tests like this recently, i.e. `T1139.hs` It makes it more traceable why a test case exists in the first place IMO